includes/Core/Authentication/Clients/OAuth_Client.php

```diff
@@ -574,7 +574,8 @@ public function get_error_message( $error_code ) {
 				$message = __( 'Unable to receive access token because of an unsupported grant type.', 'google-site-kit' );
 				break;
 			default:
-				$message = __( 'Unknown Error', 'google-site-kit' );
+				/* translators: %s: error code from API */
+				$message = sprintf( __( 'Unknown Error (code: %s).', 'google-site-kit' ), (string) $error_code );
```

Comment:
I don't think casting the `$error_code` to a string is particularly necessary here.
Out of curiosity, is there a particular case where this would make a difference?

Reply:
Fair point, I updated it. Indeed no purpose of casting, given the method signature.